templates/guides/quickstart-dhcp.md

```diff
@@ -0,0 +1,236 @@
+
+```shell
+terraform init
+```
+
+## Configuring Resources
+
+### IPAM and DHCP Resources
+In this example, we will use the following resources to create an IP space, address block, and subnet.
+
+- [bloxone_ipam_ip_space](https://registry.terraform.io/providers/infobloxopen/bloxone/latest/docs/resources/ipam_ip_space)
+- [bloxone_ipam_address_block](https://registry.terraform.io/providers/infobloxopen/bloxone/latest/docs/resources/ipam_address_block)
+- [bloxone_ipam_subnet](https://registry.terraform.io/providers/infobloxopen/bloxone/latest/docs/resources/ipam_subnet)
+
+Add the following to the `main.tf` file:
+
+````terraform
+// Create an IP space
+resource "bloxone_ipam_ip_space" "this" {
+  name = "example"
+}
+
+// Create an address block within the IP space
+resource "bloxone_ipam_address_block" "this" {
+  space   = bloxone_ipam_ip_space.this.id
+  address = "10.0.0.0"
+  cidr    = "16"
+}
+
+// Create a subnet within the address block
+resource "bloxone_ipam_subnet" "this" {
+  space   = bloxone_ipam_ip_space.this.id
+  address = "10.0.0.0"
+  cidr    = "24"
+}
+
+````
+
+We can now run `terraform plan` to see what resources will be created.
+
+```shell
+terraform plan
+```
+
+Further, we will create a range and a fixed address reservation within the subnet.
+
+We will use the following resources to create these
+- [bloxone_ipam_range](https://registry.terraform.io/providers/infobloxopen/bloxone/latest/docs/resources/ipam_range)
+- [bloxone_dhcp_fixed_address](https://registry.terraform.io/providers/infobloxopen/bloxone/latest/docs/resources/dhcp_fixed_address)
+
+We will use the following data sources to get the option codes in the default DHCPv4 option space
+- [bloxone_dhcp_option_spaces](https://registry.terraform.io/providers/infobloxopen/bloxone/latest/docs/data-sources/dhcp_option_spaces)
+- [bloxone_dhcp_option_codes](https://registry.terraform.io/providers/infobloxopen/bloxone/latest/docs/data-sources/dhcp_option_codes)
+
+Add the following code to your main.tf:
+
+````terraform
+// Get the default option space for DHCPv4
+data "bloxone_dhcp_option_spaces" "dhcp4" {
+  filters = {
+    name = "dhcp4"
+  }
+}
+
+// Get the option codes for the default option space
+data "bloxone_dhcp_option_codes" "dhcp4" {
+  filters = {
+    option_space = data.bloxone_dhcp_option_spaces.dhcp4.results.0.id
+  }
+}
+
+locals {
+  // Create a map of option code names to option code IDs
+  // This will be used to look up the option code ID by name
+  dhcp4_option_code_lookup = zipmap(data.bloxone_dhcp_option_codes.dhcp4.results[*].name, data.bloxone_dhcp_option_codes.dhcp4.results[*].id)
+}
+
+// Create a range within the subnet
+resource "bloxone_ipam_range" "this" {
+  space = bloxone_ipam_ip_space.this.id
+  start = "10.0.0.5"
+  end   = "10.0.0.100"
+  dhcp_options = [
+    {
+      option_code  = local.dhcp4_option_code_lookup["domain-name-servers"]
+      option_value = "1.1.1.1"
```

Comment:
Let's avoid promoting a competitor (cloudflare's dns address)...
Use an address within the 10/16 network.

Reply:
Changed to 10.0.0.1